code/datums/elements/basic_eating.dm

```diff
@@ -93,4 +93,5 @@
 	if(isstack(target)) //if stack, only consume 1
 		var/obj/item/stack/food_stack = target
 		final_target = food_stack.split_stack(eater, 1)
+	log_admin("[eater] has eaten [target]!")
```

Comment:
Are we sure this needs to be log_admin? maybe put it in the individual `eater`'s log_attack or something